scripts/check_latest-dev_updates.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import sys
+import re
+
+
+# also need a function to separate comments from actual commands
+def remove_comments(lines):
+    code_lines = []
+    for line in lines:
+        if not line.startswith("--"):
+            code_lines.append(line)
+    return "\n".join(code_lines)
+
+
+def unsafe_catalog_modification(s):
+    # catalog tables reside in _timescaledb_catalog, _timescaledb_internal and _timescaledb_config
+    # it is unsafe to simply ADD or DROP columns from catalog tables
+    matches = re.search(
+        r"ALTER\s*TABLE\s*(_timescaledb_catalog|_timescaledb_internal|_timescaledb_config)[\s\S]+(DROP|ADD)\s*COLUMN",
+        s,
+        flags=re.IGNORECASE,
+    )
+    if matches:
+        return True
+    return False
+
+
+def main():
+    # Open latest-dev.sql
+    latest_dev = sys.argv[1]
+    unsafe_modification = False
+    contents = ""
+
+    with open(latest_dev, "r", encoding="utf-8") as ldev:
+        # remove comments, we don't want to confuse them with actual commands
+        lines = ldev.readlines()
+        contents = remove_comments(lines)
+        unsafe_modification = unsafe_catalog_modification(contents)
+    if unsafe_modification:
+        print(
+            """ERROR: Attempting to alter timescaledb catalog tables without rebuilding the table."""
+            + """\nRebuilding catalog tables is required to ensure consistent attribute numbers across versions."""
+        )
```

Comment:
Nit: Triple quotes are used in Python if you want to define multi-line strings. So something like this works

```suggestion
        print("""
ERROR: Attempting to alter timescaledb catalog tables without rebuilding the table.
Rebuilding catalog tables is required to ensure consistent attribute numbers across versions.
"""
)
```

In this case, you don't use multi-line strings. So, simple quotes would be enough:

```suggestion
        print(
            "ERROR: Attempting to alter timescaledb catalog tables without rebuilding the table."
            + "\nRebuilding catalog tables is required to ensure consistent attribute numbers across versions."
)
```